app/helpers/__init__.py

```diff
@@ -0,0 +1,32 @@
+import os.path
+from werkzeug.exceptions import BadRequest
+
+
+def check_color_channels(r, g, b):
```

Comment:
For both, this and the "check_path"-functions, I initially defined return statements consisting of a status_code (e.g. 200 or 400) and a message. These status_codes and response messages were then evaluated within the invoking routes.py and a BadRequest was raised if necessary.

Now the BadRequest will already be raised from within the check-functions, which saves some lines of code and no return statement needs to be defined.

Not sure which way is more elegant, both will do for me.

Reply:
In flask you can use the abort() method with the code and message. This method is then used to raise the correct exception based on the code. Because we use Flask I would use the Flask way to do things and not the Werkzeug.

Reply:
This is now implemented very similar to service qrcode, using abort()